config/containerized-transfer-example.yml

```
@@ -0,0 +1,33 @@
+    identity_file: ~/.ssh/docker_prod_key
+    appdata_path: /opt/docker-data
+    description: Production server with permission-restricted files
+
+  staging:
+    hostname: staging.example.com
+    user: docker
+    identity_file: ~/.ssh/docker_staging_key
+    appdata_path: /opt/docker-data
+    description: Staging server
+
+# Environment variables can also be used:
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)